qless/workers/forking.py

```diff
@@ -33,7 +38,7 @@ def stop(self, sig=signal.SIGINT):
                 logger.exception('Error stopping %s...' % cpid)
 
         # While we still have children running, wait for them
-        for cpid in self.sandboxes.keys():
+        for cpid in list(self.sandboxes):
```

Comment:
Ah, I see. That's subtle. Let's add a comment about the in-place modification, but still use the normal idiom in the other places.